.github/workflows/inspektor-gadget.yml

```diff
@@ -872,6 +874,20 @@ jobs:
           hash=$(sha256sum $i | cut -d' ' -f1)
           echo "${hash}  $(basename $i)" >> inspektor-gadget-${{ github.ref_name }}_checksums.txt
         done
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+      with:
+        # We need to cosign v2.0.0 to use keyless signing without using
+        # COSIGN_EXPERIMENTAL. This version was released recently:
+        # https://blog.sigstore.dev/cosign-2-0-released/
```

Comment:
Could you please update the PR description? I see it is using the `COSIGN_EXPERIMENTAL` and the signature files (which I are not used anymore). Thanks!